crates/cheatcodes/src/inspector.rs

```diff
@@ -348,9 +408,13 @@ impl Cheatcodes {
     }
 }
 
-impl<DB: DatabaseExt> Inspector<DB> for Cheatcodes {
+impl Cheatcodes {
```

Comment:
what's stopping this from keeping the inspector implementation?

Reply:
`call` hook signature is now different